set.mm

```diff
@@ -70390,11 +70413,11 @@ result of an operator (deduction version).  (Contributed by Paul
     ovigg.1 $e |- ( ( x = A /\ y = B /\ z = C ) -> ( ph <-> ps ) ) $.
     ovigg.4 $e |- E* z ph $.
     ovigg.5 $e |- F = { <. <. x , y >. , z >. | ph } $.
-    $( The value of an operation class abstraction.  Compare ~ ovig .  The
-       condition ` ( x e. R /\ y e. S ) ` is been removed.  (Contributed by FL,
+    $( The value of an operation class abstraction.  Compared with ~ ovig , the
+       condition ` ( x e. R /\ y e. S ) ` is removed.  (Contributed by FL,
        24-Mar-2007.)  (Revised by Mario Carneiro, 19-Dec-2013.) $)
-    ovigg $p |- ( ( A e. V /\ B e. W /\ C e. X ) ->
-                         ( ps -> ( A F B ) = C ) ) $=
+    ovigg $p |-
+               ( ( A e. V /\ B e. W /\ C e. X ) -> ( ps -> ( A F B ) = C ) ) $=
```

Comment:
I find some of these reformattings a bit dubious. I'm very much not a fan of right alignment, and would prefer we used a consistent formatting style in the library.

Reply:
I can revert that.  I thought the rule of thumb was to break line at highest possible token (in the sense of "oldest in the syntax tree), and I would actually break these at `$p` if not for formatting problems arising from that choice.

Consistency is indeed paramount.  Actually, I wouldn't mind having an automatic "dumb" line-wrap as for proofs and comments.